bus-mapping/src/circuit_input_builder.rs

```diff
@@ -320,7 +329,12 @@ impl<C: CircuitsParams> CircuitInputBuilder<C> {
         // accumulates gas across all txs in the block
         for (tx_index, tx) in eth_block.transactions.iter().enumerate() {
             let geth_trace = &geth_traces[tx_index];
-            self.handle_tx(tx, geth_trace, tx_index + 1 == eth_block.transactions.len())?;
+            self.handle_tx(
```

Comment:
I added a new variable and a comment.
As I understand it, `tx_index + 1` is not the "next" tx id. The  `tx_index + 1` is there because the tx id starts from 1.